app/code/Magento/NewRelicReporting/Test/Mftf/Test/AdminChecksSystemConfigDependencyTest.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminChecksSystemConfigDependencyTest">
+        <annotations>
+            <features value="NewRelicReporting"/>
+            <stories value="Admin is able to see the configuration fields only after enabling the feature"/>
+            <title value="Admin can see the configuration fields only after enabling the feature"/>
+            <description value="The system configs should be available only after enabling the New Relic feature."/>
+            <severity value="CRITICAL"/>
+            <group value="NewRelicReporting"/>
+        </annotations>
+        <before>
+            <actionGroup ref="LoginAsAdmin" stepKey="loginAsAdmin"/>
+            <actionGroup ref="AdminNavigateToNewRelicConfigurationActionGroup" stepKey="goToConfigPage"/>
+            <actionGroup ref="AdminExpandNewRelicConfigTabActionGroup" stepKey="expandingGeneralTab">
```

Comment:
Please use generic action group for expanding config tabs: https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Config/Test/Mftf/ActionGroup/AdminExpandConfigTabActionGroup.xml

Here is an example how this action group is used in tests: https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Backend/Test/Mftf/Test/AdminCheckLocaleAndDeveloperConfigInProductionModeTest.xml#L37-L39

Reply:
Looks like the ActionGroup mentioned by you is expanding tabs (from the left navigations) and not sections, so I've implemented a new ActionGroup for this matter.